projects/assessment-summary-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/Handler.kt

```diff
@@ -21,14 +23,27 @@ class Handler(
     override val converter: NotificationConverter<HmppsDomainEvent>,
     private val ordsClient: OrdsClient,
     private val assessmentSubmitted: AssessmentSubmitted,
-    private val featureFlags: FeatureFlags
+    private val featureFlags: FeatureFlags,
+    private val telemetryService: TelemetryService
 ) : NotificationHandler<HmppsDomainEvent> {
     @Publish(messages = [Message(messageId = AssessmentSummaryProduced, payload = Schema(HmppsDomainEvent::class))])
     override fun handle(notification: Notification<HmppsDomainEvent>) {
-        if (notification.message.eventType == AssessmentSummaryProduced && featureFlags.enabled("assessment-summary-produced")) {
-            notification.message.detailUrl
-                ?.let { ordsClient.getAssessmentSummary(URI.create(it)) }
-                ?.let { assessmentSubmitted.assessmentSubmitted(it.crn, it.assessments.first()) }
+        try {
+            if (notification.message.eventType == AssessmentSummaryProduced && featureFlags.enabled("assessment-summary-produced")) {
+                notification.message.detailUrl
+                    ?.let { ordsClient.getAssessmentSummary(URI.create(it)) }
+                    ?.let { assessmentSubmitted.assessmentSubmitted(it.crn, it.assessments.first()) }
+            }
+        } catch (ime: IgnorableMessageException) {
+            telemetryService.trackEvent(
+                "AssessmentSummaryFailure",
```

Comment:
Could we change this to something like ignored, rather than failed?

```suggestion
                "AssessmentSummaryIgnored",
```